app/Handlers/Events/SendFirstVoteMessage.php

```diff
@@ -73,8 +62,8 @@ public function handle(UserCastFirstVote $event)
             ];
         }
 
-        $payload = serialize($payload);
-        $broker->publishMessage($payload);
+        $routingKey = env('VOTE_ROUTING_KEY', 'votingapp.event.vote');
+        $this->broker->publish('vote', $payload, $routingKey);
```

Comment:
Ha, you beat me to it! I'm in the process of updating `publishMessage()` method in Message Broker to use the routingKey value as a parm.